src/plumbing/fnk/impl.clj

```diff
@@ -37,6 +37,16 @@
   [k]
   (symbol (name k)))
 
+(defn qualified-k->sym
+  "Make a keyword into a symbol, preserving namespace"
+  [k]
+  (symbol (namespace k) (name k)))
```

Comment:
> That can just be fixed to work on qualified symbols rather than adding a new fn i think.

I looked into that, but `k->sym` is used in places where qualified symbol isn't desired. For example

```clojure
(p/fnk [a/b b/c c/d] [b c d])
```

Expands to

```clojure
(clojure.core/let [pos-fn__62754__auto__ (clojure.core/fn 
                                           fnk64148-positional
                                           [d64149 c64150 b64151]
                                           (clojure.core/let 
                                             [b b64151]
                                             (clojure.core/let 
                                               [c c64150]
                                               (clojure.core/let 
                                                 [d d64149]
                                                 (do [b c d])))))]
  (clojure.core/vary-meta
    (schema.core/fn
      fnk64148
      [m__62755__auto__
       :-
       {schema.core/Keyword schema.core/Any,
        :c/d schema.core/Any,
        :b/c schema.core/Any,
        :a/b schema.core/Any}]
      (plumbing.core/letk
        [[c/d b/c a/b] m__62755__auto__]
        (pos-fn__62754__auto__ c/d b/c a/b)))       ;; <-- No such namepsace: c, b, a
    clojure.core/assoc
    :plumbing.fnk.impl/positional-info
    [pos-fn__62754__auto__ [:c/d :b/c :a/b]]))
```

I'll think about a way to consolidate these 3 functions